.github/workflows/coding_style_checks.yml

```diff
@@ -0,0 +1,26 @@
+name: Run coding style check
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  format:
+    name: Check code formatting
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+    steps:
+      - name: Clone
+        uses: actions/checkout@v2
+      - name: Run cargo fmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
+    
```

Comment:
do we want to make this workflow reusable (as per C BP) ?

Reply:
Done.